### PR TITLE
dumpling: add URI formats

### PR DESCRIPTION
**Pull request by hfxsd**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

- Created a new URI file.
- Updated all releated links to the new URI file.
- Add Azure Blob Storage in TiDB Lightning.
- Fix a typo https://github.com/pingcap/docs/issues/15172

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [ ] v7.5 (TiDB 7.5 versions)
- [x] v7.4 (TiDB 7.4 versions)
- [x] v7.3 (TiDB 7.3 versions)
- [x] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:https://github.com/pingcap/docs-cn/pull/15289
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: dumpling-overview.md [hunk 1]
```diff
@@ -116,6 +104,62 @@ In the command above:
...
+<SimpleTab groupId="storage">
+<div label="Amazon S3" value="amazon">
+
+- `scheme`: `s3`
+- `host`: `bucket name`
+- `parameters`:
+
+    - `access-key`: Specifies the access key.
+    - `secret-access-key`: Specifies the secret access key.
+    - `session-token`: Specifies the temporary session token. BR does not support this parameter yet.
```
Comment (Frank945946) [inline dumpling-overview.md:124]: 这部分也待确认

Comment (hfxsd) [inline dumpling-overview.md:124]:
```suggestion
    - `session-token`: Specifies the temporary session token.
```

Comment (hfxsd) [reply dumpling-overview.md]: 删掉了 BR does not support this parameter yet. 这句话

File: sql-statements/sql-statement-restore.md [hunk 2]
```diff
@@ -103,17 +103,7 @@ BR supports restoring data from S3 or GCS:
...
-The URL syntax is further explained in [external storage URI](/br/backup-and-restore-storages.md#uri-format).
-
-</CustomContent>
-
-<CustomContent platform="tidb-cloud">
-
-The URL syntax is further explained in [external storage URI](https://docs.pingcap.com/tidb/stable/backup-and-restore-storages#uri-format).
-
-</CustomContent>
+The URL syntax is further explained in [external storage URI](/external-storage-uri.md).
```
Comment (hfxsd) [inline sql-statements/sql-statement-restore.md:106]:
```suggestion
The URI syntax is further explained in [external storage URI](/external-storage-uri.md).
```

File: system-variables.md [hunk 3]
```diff
@@ -1580,18 +1580,18 @@ mysql> SELECT job_info FROM mysql.analyze_jobs ORDER BY end_time DESC LIMIT 1;
 
 <CustomContent platform="tidb">
 
-- This variable is used to specify the cloud storage URI to enable [Global Sort](/tidb-global-sort.md). After enabling the [distributed execution framework](/tidb-distributed-execution-framework.md), you can use the Global Sort feature by configuring the URI and pointing it to an appropriate cloud storage path with the necessary permissions to access the storage. For more details, see [URI format](/br/backup-and-restore-storages.md#uri-format).
+- This variable is used to specify the cloud storage URI to enable [Global Sort](/tidb-global-sort.md). After enabling the [distributed execution framework](/tidb-distributed-execution-framework.md), you can use the Global Sort feature by configuring the URI and pointing it to an appropriate cloud storage path with the necessary permissions to access the storage. For more details, see [URI formats](/external-storage-uri.md).
```
Comment (hfxsd) [inline system-variables.md:1583]:
```suggestion
- This variable is used to specify the cloud storage URI to enable [Global Sort](/tidb-global-sort.md). After enabling the [distributed execution framework](/tidb-distributed-execution-framework.md), you can use the Global Sort feature by configuring the URI and pointing it to an appropriate cloud storage path with the necessary permissions to access the storage. For more details, see [URI Formats of External Storage Services](/external-storage-uri.md).
```

File: system-variables.md [hunk 4]
```diff
@@ -1580,18 +1580,18 @@ mysql> SELECT job_info FROM mysql.analyze_jobs ORDER BY end_time DESC LIMIT 1;
...
     - The [`IMPORT INTO`](/sql-statements/sql-statement-import-into.md) statement for import jobs of TiDB Self-Hosted. For TiDB Cloud, the `IMPORT INTO` statement is not applicable.
 
 </CustomContent>
 <CustomContent platform="tidb-cloud">
 
-- This variable is used to specify the cloud storage URI to enable [Global Sort](/tidb-global-sort.md). After enabling the [distributed execution framework](/tidb-distributed-execution-framework.md), you can use the Global Sort feature by configuring the URI and pointing it to an appropriate cloud storage path with the necessary permissions to access the storage. For more details, see [URI format](https://docs.pingcap.com/tidb/stable/backup-and-restore-storages#uri-format).
+- This variable is used to specify the cloud storage URI to enable [Global Sort](/tidb-global-sort.md). After enabling the [distributed execution framework](/tidb-distributed-execution-framework.md), you can use the Global Sort feature by configuring the URI and pointing it to an appropriate cloud storage path with the necessary permissions to access the storage. For more details, see [URI formats](/external-storage-uri.md).
```
Comment (hfxsd) [inline system-variables.md:1591]:
```suggestion
- This variable is used to specify the cloud storage URI to enable [Global Sort](/tidb-global-sort.md). After enabling the [distributed execution framework](/tidb-distributed-execution-framework.md), you can use the Global Sort feature by configuring the URI and pointing it to an appropriate cloud storage path with the necessary permissions to access the storage. For more details, see [URI Formats of External Storage Services](/external-storage-uri.md).
```

File: external-storage-uri.md [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+
+## Amazon S3 URI format
+
+- `scheme`: `s3`
+- `host`: `bucket name`
+- `parameters`:
+
+    - `access-key`: Specifies the access key.
+    - `secret-access-key`: Specifies the secret access key.
+    - `session-token`: Specifies the temporary session token.
```
Comment (hfxsd) [inline external-storage-uri.md:22]:
```suggestion
    - `session-token`: Specifies the temporary session token. BR does not support this parameter yet.
```

File: system-variables.md [hunk 6]
```diff
@@ -1580,18 +1580,18 @@ mysql> SELECT job_info FROM mysql.analyze_jobs ORDER BY end_time DESC LIMIT 1;
 
 <CustomContent platform="tidb">
 
-- This variable is used to specify the cloud storage URI to enable [Global Sort](/tidb-global-sort.md). After enabling the [distributed execution framework](/tidb-distributed-execution-framework.md), you can use the Global Sort feature by configuring the URI and pointing it to an appropriate cloud storage path with the necessary permissions to access the storage. For more details, see [URI format](/br/backup-and-restore-storages.md#uri-format).
+- This variable is used to specify the cloud storage URI to enable [Global Sort](/tidb-global-sort.md). After enabling the [distributed execution framework](/tidb-distributed-execution-framework.md), you can use the Global Sort feature by configuring the URI and pointing it to an appropriate cloud storage path with the necessary permissions to access the storage. For more details, see [URI Formats of External Storage Services](/external-storage-uri.md).
```
Comment (hfxsd) [inline system-variables.md:1583]:
```suggestion
- This variable is used to specify the Amazon S3 cloud storage URI to enable [Global Sort](/tidb-global-sort.md). After enabling the [distributed execution framework](/tidb-distributed-execution-framework.md), you can use the Global Sort feature by configuring the URI and pointing it to an appropriate cloud storage path with the necessary permissions to access the storage. For more details, see [Amazon S3 URI Format](/external-storage-uri.md#amazon-s3-uri-format).
```

File: tidb-lightning/tidb-lightning-overview.md [hunk 7]
```diff
@@ -17,8 +17,8 @@ TiDB Lightning supports the following file formats:
 TiDB Lightning can read data from the following sources:
 
 - Local
-- [Amazon S3](/br/backup-and-restore-storages.md#uri-format)
-- [Google Cloud Storage](/br/backup-and-restore-storages.md#uri-format)
+- [Amazon S3](/external-storage-uri.md#amazon-s3-uri-format)
+- [Google Cloud Storage](/external-storage-uri.md#gcs-uri-format)
```
Comment (hfxsd) [inline tidb-lightning/tidb-lightning-overview.md:21]:
```suggestion
- [Google Cloud Storage](/external-storage-uri.md#gcs-uri-format)
- [Azure Blob Storage](/external-storage-uri.md#azure-blob-storage-uri-format)
```

File: sql-statements/sql-statement-import-into.md [hunk 8]
```diff
@@ -247,10 +247,16 @@ IMPORT INTO t FROM '/path/to/file-*.csv'
...
     ```sql
-    IMPORT INTO t FROM 'gs://bucket-name/test.csv';
+    IMPORT INTO t FROM 'gs://bucket-name/test.csv?credentials-file=${credentials-file-path}';
     ```
 
-For details about the URI path configuration for Amazon S3 or GCS, see [External storage](/br/backup-and-restore-storages.md#uri-format).
+- Import data files from Azure Blob Storage:
+
+    ```sql
+    IMPORT INTO t FROM ''azure://bucket-name/test.csv?credentials-file=${credentials-file-path}';';
```
Comment (hfxsd) [inline sql-statements/sql-statement-import-into.md:256]:
```suggestion
    IMPORT INTO t FROM 'azure://external/backup-20220915?account-name=${account-name}&account-key=${account-key}';
```

File: TOC-tidb-cloud.md [hunk 9]
```diff
@@ -620,6 +620,7 @@
   - [TiDB Global Sort](/tidb-global-sort.md)
   - [DDL Execution Principles and Best Practices](/ddl-introduction.md)
   - [Troubleshoot Inconsistency Between Data and Indexes](/troubleshoot-data-inconsistency-errors.md)
+  - [URI Formats](/external-storage-uri.md)
```
Comment (hfxsd) [inline TOC-tidb-cloud.md:623]:
```suggestion
  - [URI Formats of External Storage Services](/external-storage-uri.md)
```

File: TOC.md [hunk 10]
```diff
@@ -995,6 +995,7 @@
   - [Error Codes](/error-codes.md)
   - [Table Filter](/table-filter.md)
   - [Schedule Replicas by Topology Labels](/schedule-replicas-by-topology-labels.md)
+  - [URI Formats](/external-storage-uri.md)
```
Comment (hfxsd) [inline TOC.md:998]:
```suggestion
  - [URI Formats of External Storage Services](/external-storage-uri.md)
```

File: dumpling-overview.md [hunk 11]
```diff
@@ -233,19 +231,7 @@ export AWS_ACCESS_KEY_ID=${AccessKey}
...
-</CustomContent>
-
-<CustomContent platform="tidb-cloud">
-
-Dumpling also supports reading credential files from `~/.aws/credentials`. Parameters for exporting data to Amazon S3 using Dumpling are the same as the parameters used in BR. For more parameter descriptions, see [external storage URI](https://docs.pingcap.com/tidb/stable/backup-and-restore-storages#uri-format).
-
-</CustomContent>
-
-{{< copyable "shell-regular" >}}
+Dumpling also supports reading credential files from `~/.aws/credentials`. Parameters for exporting data to Amazon S3 using Dumpling are the same as the parameters used in BR. For more parameter descriptions, see [external storage URI](/external-storage-uri.md).
```
Comment (hfxsd) [inline dumpling-overview.md:234]:
```suggestion
Dumpling also supports reading credential files from `~/.aws/credentials`. For more URI parameter descriptions, see [external storage URI](/external-storage-uri.md).
```

File: sql-statements/sql-statement-import-into.md [hunk 12]
```diff
@@ -247,10 +247,16 @@ IMPORT INTO t FROM '/path/to/file-*.csv'
 - Import data files from GCS:
 
     ```sql
-    IMPORT INTO t FROM 'gs://bucket-name/test.csv';
+    IMPORT INTO t FROM 'gs://bucket-name/test.csv?credentials-file=${credentials-file-path}';
```
Comment (hfxsd) [inline sql-statements/sql-statement-import-into.md:250]:
```suggestion
    IMPORT INTO t FROM 'gs://import/test.csv?credentials-file=${credentials-file-path}';
```

File: ticdc/ticdc-sink-to-cloud-storage.md [hunk 13]
```diff
@@ -79,7 +79,7 @@ The following is an example configuration for Azure Blob Storage:
 
 > **Tip:**
 >
-> The URI parameters of Amazon S3, GCS, and Azure Blob Storage in TiCDC are the same as their URI parameters in BR. For details, see [Backup storage URI format](/br/backup-and-restore-storages.md#uri-format-description).
+> The URI parameters of Amazon S3, GCS, and Azure Blob Storage in TiCDC are the same as their URI parameters in BR. For details, see [URI Formats of External Storage Services](/external-storage-uri.md).
```
Comment (hfxsd) [inline ticdc/ticdc-sink-to-cloud-storage.md:82]:
```suggestion
> For more information about the URI parameters of Amazon S3, GCS, and Azure Blob Storage in TiCDC, see [URI Formats of External Storage Services](/external-storage-uri.md).
```

File: ticdc/ticdc-sink-to-kafka.md [hunk 14]
```diff
@@ -389,7 +389,7 @@ When `large-message-handle-option` is set to `"claim-check"`, `claim-check-stora
 
 > **Tip**
 >
-> Currently, the external storage services supported by TiCDC are the same as BR. For detailed parameter descriptions, see [Backup storages URI format](/br/backup-and-restore-storages.md#uri-format-description).
+> Currently, the external storage services supported by TiCDC are the same as BR. For detailed parameter descriptions, see [URI Formats of External Storage Services](/external-storage-uri.md).
```
Comment (hfxsd) [inline ticdc/ticdc-sink-to-kafka.md:392]:
```suggestion
> For more information about the URI parameters of Amazon S3, GCS, and Azure Blob Storage in TiCDC, see [URI Formats of External Storage Services](/external-storage-uri.md).
```

File: sql-statements/sql-statement-import-into.md [hunk 15]
```diff
@@ -137,7 +137,7 @@ The supported options are described as follows:
...
+| `CLOUD_STORAGE_URI` | All formats | Specifies the target address where encoded KV data for [global sorting](#global-sorting) is stored. When `CLOUD_STORAGE_URI` is not specified, `IMPORT INTO` determines whether to use global sorting based on the value of the system variable [`tidb_cloud_storage_uri`](/system-variables.md#tidb_cloud_storage_uri-new-in-v740). If this system variable specifies a target storage address, `IMPORT INTO` uses this address for global sorting. When `CLOUD_STORAGE_URI` is specified with a non-empty value, `IMPORT INTO` uses that value as the target storage address. When `CLOUD_STORAGE_URI` is specified with an empty value, local sorting is enforced. Currently, the target storage address only supports S3. For details about the URI configuration, see [Amazon S3 URI format](/external-storage-uri.md#amazon-s3-uri--format). When this feature is used, all TiDB nodes must have read and write access for the target S3 bucket. |
```
Comment (hfxsd) [inline sql-statements/sql-statement-import-into.md:140]:
```suggestion
| `CLOUD_STORAGE_URI` | All formats | Specifies the target address where encoded KV data for [global sorting](#global-sorting) is stored. When `CLOUD_STORAGE_URI` is not specified, `IMPORT INTO` determines whether to use global sorting based on the value of the system variable [`tidb_cloud_storage_uri`](/system-variables.md#tidb_cloud_storage_uri-new-in-v740). If this system variable specifies a target storage address, `IMPORT INTO` uses this address for global sorting. When `CLOUD_STORAGE_URI` is specified with a non-empty value, `IMPORT INTO` uses that value as the target storage address. When `CLOUD_STORAGE_URI` is specified with an empty value, local sorting is enforced. Currently, the target storage address only supports S3. For details about the URI configuration, see [Amazon S3 URI format](/external-storage-uri.md#amazon-s3-uri-format). When this feature is used, all TiDB nodes must have read and write access for the target S3 bucket. |
```

**Comment by lilin90** [comment]
@Frank945946 Please help confirm whether it's appropriate to add `external-storage-uri.md` to `TOC-tidb-cloud.md` too. That is, to confirm whether the "URI Formats of External Storage Services" doc also applies to TiDB Cloud. Thanks!

File: TOC-tidb-cloud.md [hunk 16]
```diff
@@ -620,6 +620,7 @@
   - [TiDB Global Sort](/tidb-global-sort.md)
   - [DDL Execution Principles and Best Practices](/ddl-introduction.md)
   - [Troubleshoot Inconsistency Between Data and Indexes](/troubleshoot-data-inconsistency-errors.md)
+  - [URI Formats of External Storage Services](/external-storage-uri.md)
```
Comment (hfxsd) [inline TOC-tidb-cloud.md:623]:
```suggestion
```

File: system-variables.md [hunk 17]
```diff
@@ -1580,18 +1580,18 @@ mysql> SELECT job_info FROM mysql.analyze_jobs ORDER BY end_time DESC LIMIT 1;
...
     - The [`IMPORT INTO`](/sql-statements/sql-statement-import-into.md) statement for import jobs of TiDB Self-Hosted. For TiDB Cloud, the `IMPORT INTO` statement is not applicable.
 
 </CustomContent>
 <CustomContent platform="tidb-cloud">
 
-- This variable is used to specify the cloud storage URI to enable [Global Sort](/tidb-global-sort.md). After enabling the [distributed execution framework](/tidb-distributed-execution-framework.md), you can use the Global Sort feature by configuring the URI and pointing it to an appropriate cloud storage path with the necessary permissions to access the storage. For more details, see [URI format](https://docs.pingcap.com/tidb/stable/backup-and-restore-storages#uri-format).
+- This variable is used to specify the cloud storage URI to enable [Global Sort](/tidb-global-sort.md). After enabling the [distributed execution framework](/tidb-distributed-execution-framework.md), you can use the Global Sort feature by configuring the URI and pointing it to an appropriate cloud storage path with the necessary permissions to access the storage. For more details, see [URI Formats of External Storage Services](/external-storage-uri.md).
```
Comment (qiancai) [inline system-variables.md:1591]:
```suggestion
- This variable is used to specify the cloud storage URI to enable [Global Sort](/tidb-global-sort.md). After enabling the [distributed execution framework](/tidb-distributed-execution-framework.md), you can use the Global Sort feature by configuring the URI and pointing it to an appropriate cloud storage path with the necessary permissions to access the storage. For more details, see [URI Formats of External Storage Services](https://docs.pingcap.com/tidb/stable/external-storage-uri).
```

File: sql-statements/sql-statement-restore.md [hunk 18]
```diff
@@ -103,17 +103,7 @@ BR supports restoring data from S3 or GCS:
...
-The URL syntax is further explained in [external storage URI](/br/backup-and-restore-storages.md#uri-format).
-
-</CustomContent>
-
-<CustomContent platform="tidb-cloud">
-
-The URL syntax is further explained in [external storage URI](https://docs.pingcap.com/tidb/stable/backup-and-restore-storages#uri-format).
-
-</CustomContent>
+The URI syntax is further explained in [URI Formats of External Storage Services](/external-storage-uri.md).
```
Comment (qiancai) [inline sql-statements/sql-statement-restore.md:106]: As `sql-statements/sql-statement-import-into.md` is also used by TiDB Cloud TOC, please use the CustomContent tag for this paragraph.

File: sql-statements/sql-statement-backup.md [hunk 19]
```diff
@@ -112,17 +112,7 @@ BR supports backing up data to S3 or GCS:
...
-The URL syntax is further explained in [external storage URI](/br/backup-and-restore-storages.md#uri-format).
-
-</CustomContent>
-
-<CustomContent platform="tidb-cloud">
-
-The URL syntax is further explained in [external storage URI](https://docs.pingcap.com/tidb/stable/backup-and-restore-storages#uri-format).
-
-</CustomContent>
+The URI syntax is further explained in [URI Formats of External Storage Services](/external-storage-uri.md).
```
Comment (qiancai) [inline sql-statements/sql-statement-backup.md:115]: As `sql-statements/sql-statement-import-into.md` is also used by TiDB Cloud TOC, please use the CustomContent tag for this paragraph.

Comment (hfxsd) [inline sql-statements/sql-statement-backup.md:115]:
```suggestion
<CustomContent platform="tidb">

The URL syntax is further explained in [external storage URI](/br/backup-and-restore-storages.md#uri-format).

</CustomContent>

<CustomContent platform="tidb-cloud">

The URL syntax is further explained in [external storage URI](https://docs.pingcap.com/tidb/stable/external-storage-uri).

</CustomContent>
```

File: sql-statements/sql-statement-restore.md (hunk 18 above)
Comment (hfxsd) [inline sql-statements/sql-statement-restore.md:106]:
```suggestion
<CustomContent platform="tidb">

The URL syntax is further explained in [external storage URI](/br/backup-and-restore-storages.md#uri-format).

</CustomContent>

<CustomContent platform="tidb-cloud">

The URL syntax is further explained in [external storage URI](https://docs.pingcap.com/tidb/stable/external-storage-uri).

</CustomContent>
```

**Review by lilin90 — COMMENTED** [review]
/lgtm

File: external-storage-uri.md [hunk 20]
```diff
@@ -0,0 +1,74 @@
...
+    - `external-id`: When you access Amazon S3 data from a third party, you might need to specify a correct [external ID](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_create_for-user_externalid.html) to assume [the IAM role](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles.html). In this case, you can use this `external-id` URL query parameter to specify the external ID and make sure that you can assume the IAM role. An external ID is an arbitrary string provided by the third party together with the IAM role ARN to access the Amazon S3 data. Providing an external ID is optional when assuming an IAM role, which means if the third party does not require an external ID for the IAM role, you can assume the IAM role and access the corresponding Amazon S3 data without providing this parameter.
+
+Example:
+
+```shell
+s3://external/backup-20220915?access-key=${access-key}&secret-access-key=${secret-access-key}
```
Comment (hfxsd) [inline external-storage-uri.md:38]:
```suggestion
s3://external/test?access-key=${access-key}&secret-access-key=${secret-access-key}
```

File: external-storage-uri.md [hunk 21]
```diff
@@ -0,0 +1,74 @@
...
+- `parameters`:
+
+    - `credentials-file`: Specifies the path to the credentials JSON file on the migration tool node.
+    - `storage-class`: Specifies the storage class of the uploaded objects (for example, `STANDARD` or `COLDLINE`)
+    - `predefined-acl`: Specifies the predefined ACL of the uploaded objects (for example, `private` or `project-private`)
+
+Example:
+
+```shell
+gcs://external/backup-20220915?credentials-file=${credentials-file-path}
```
Comment (hfxsd) [inline external-storage-uri.md:54]:
```suggestion
gcs://external/test?credentials-file=${credentials-file-path}
```

File: external-storage-uri.md [hunk 22]
```diff
@@ -0,0 +1,74 @@
...
+    - `account-key`: Specifies the access key.
+    - `sas-token`: Specifies the shared access signature (SAS) token.
+    - `access-tier`: Specifies the access tier of the uploaded objects, for example, `Hot`, `Cool`, or `Archive`. The default value is the default access tier of the storage account.
+    - `encryption-scope`: Specifies the [encryption scope](https://learn.microsoft.com/en-us/azure/storage/blobs/encryption-scope-manage?tabs=powershell#upload-a-blob-with-an-encryption-scope) for server-side encryption.
+    - `encryption-key`: Specifies the [encryption key](https://learn.microsoft.com/en-us/azure/storage/blobs/encryption-customer-provided-keys) for server-side encryption, which uses the AES256 encryption algorithm.
+
+Example:
+
+```shell
+azure://external/backup-20220915?account-name=${account-name}&account-key=${account-key}
```
Comment (hfxsd) [inline external-storage-uri.md:73]:
```suggestion
azure://external/test.csv?account-name=${account-name}&account-key=${account-key}
```

File: external-storage-uri.md [hunk 23]
```diff
@@ -0,0 +1,92 @@
...
+
+The following is an example of an Amazon S3 URI for TiDB Lightning and BR. In this example, you need to specify a specific file path `testfolder`.
+
+```shell
+s3://external/testfolder?access-key=${access-key}&secret-access-key=${secret-access-key}
+```
+
+The following is an example of an Amazon S3 URI for [IMPORT INTO](/sql-statements/sql-statement-import-into.md). In this example, you need to specify a specific filename `test.csv`.
```
Comment (lilin90) [inline external-storage-uri.md:41]:
```suggestion
The following is an example of an Amazon S3 URI for [`IMPORT INTO`](/sql-statements/sql-statement-import-into.md). In this example, you need to specify a specific filename `test.csv`.
```

File: external-storage-uri.md [hunk 24]
```diff
@@ -0,0 +1,92 @@
...
+    - `storage-class`: Specifies the storage class of the uploaded objects (for example, `STANDARD` or `COLDLINE`)
+    - `predefined-acl`: Specifies the predefined ACL of the uploaded objects (for example, `private` or `project-private`)
+
+The following is an example of a GCS URI for TiDB Lightning and BR. In this example, you need to specify a specific file path `testfolder`.
+
+```shell
+gcs://external/testfolder?credentials-file=${credentials-file-path}
+```
+
+The following is an example of a GCS URI for [IMPORT INTO](/sql-statements/sql-statement-import-into.md). In this example, you need to specify a specific filename `test.csv`.
```
Comment (lilin90) [inline external-storage-uri.md:63]:
```suggestion
The following is an example of a GCS URI for [`IMPORT INTO`](/sql-statements/sql-statement-import-into.md). In this example, you need to specify a specific filename `test.csv`.
```

File: external-storage-uri.md [hunk 25]
```diff
@@ -0,0 +1,92 @@
...
+    - `encryption-scope`: Specifies the [encryption scope](https://learn.microsoft.com/en-us/azure/storage/blobs/encryption-scope-manage?tabs=powershell#upload-a-blob-with-an-encryption-scope) for server-side encryption.
+    - `encryption-key`: Specifies the [encryption key](https://learn.microsoft.com/en-us/azure/storage/blobs/encryption-customer-provided-keys) for server-side encryption, which uses the AES256 encryption algorithm.
+
+The following is an example of an Azure Blob Storage URI for TiDB Lightning and BR. In this example, you need to specify a specific file path `testfolder`.
+
+```shell
+azure://external/testfolder?account-name=${account-name}&account-key=${account-key}
+```
+
+The following is an example of an Azure Blob Storage URI for [IMPORT INTO](/sql-statements/sql-statement-import-into.md). In this example, you need to specify a specific filename `test.csv`.
```
Comment (lilin90) [inline external-storage-uri.md:88]:
```suggestion
The following is an example of an Azure Blob Storage URI for [`IMPORT INTO`](/sql-statements/sql-statement-import-into.md). In this example, you need to specify a specific filename `test.csv`.
```

**Review by lilin90 — COMMENTED** [review]
LGTM

**Pull request merged**